RecoEgamma/ElectronIdentification/python/ElectronMVAValueMapProducer_cfi.py

```diff
@@ -15,6 +15,13 @@
 from RecoEgamma.ElectronIdentification.Identification.mvaElectronID_Spring15_25ns_Trig_V1_cff import *
 mvaConfigsForEleProducer.append( mvaEleID_Spring15_25ns_Trig_V1_producer_config )
 
+from RecoEgamma.ElectronIdentification.Identification.mvaElectronID_Spring16_HZZ_V1_cff import *
```

Comment:
Please don't use `import *`. Please list the modules you need to import. There are many instances of `import *` in this PR that should be fixed.